Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -243,6 +230,15 @@ void HoI4::World::addNeutrality()
 }
 
 
+void HoI4::World::addLeaders()
+{
+	for (auto country: countries)
```

Comment:
copy is fine?

Reply:
While pointers are involved, yes. But it would cause problems if those are removed, plus is unclear, so changed.